### DISCUSSION TITLE
Should box-shadow be for screen only?

### DISCUSSION THREAD

**Discussion by taasan** [picocss/pico#285] [General]
First of all, thank you for creating this wonderful framework.

Box shadows (for cards at least) doesn't work well on print media imo.  They are easily removed, but should maybe be off by default.

## Example

![image](https://user-images.githubusercontent.com/16260757/208897883-8be46950-217e-44ef-a2b5-8090fa0d51d2.png)

**Comment by lucaslarroche** [comment]
Hi @taasan, 

That is a specific case. In most cases, `<article />` with shadows looks good when printed.

If we remove the shadows on the articles for print, they will no longer stand out from the rest of the content, and more importantly, there would be an inner margin that would not make sense (the content would appear unaligned for no reason).

I am not in favor of making these changes globally.
I think it should be done on a case-by-case basis with custom CSS code.

For example, you could avoid a page break:

```CSS
article {
  break-inside: avoid;
}
```

Or replace the shadows with a border.
